R/Lab 1.Rmd

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
The paths to the images should be set using the system.file function. https://stat.ethz.ch/R-manual/R-devel/library/base/html/system.file.html

If you put the images in inst/lab1/images/, you would do something like: `imagepath <- system.file('lab1/images/figure1.jpg',package='NWCEd')` in your code. This will work because all the stuff in 'inst' gets installed into the base folder of the package and `system.file('./', package='NWCEd')` will return the install location of the package.